cmd/clusterctl/cmd/describe_cluster_test.go

```diff
@@ -261,31 +278,30 @@ func Test_TreePrefix(t *testing.T) {
 				return obectjTree
 			}(),
 			expectPrefix: []string{
-				"Object/root",
-				"├─Object/child1",
-				"│ │           ├─C1.1", // first condition child gets pipes, children pipe and ├─
-				"│ │           └─C1.2", // last condition child gets pipes, children pipe and └─
-				"│ └─Object/child1.1",
+				`"Object/root"
+				"├─Object/child1"
+				"│ │           ├─C1.1"
+				"│ │           └─C1.2"
+				"│ └─Object/child1.1"
 				"└─Object/child2",
-				"  │           ├─C2.1", // first condition child gets spaces, children pipe and ├─
-				"  │           └─C2.2", // last condition child gets spaces, children pipe and └─
-				"  └─Object/child2.1",
+				"  │           ├─C2.1"
+				"  │           └─C2.2"
+				"  └─Object/child2.1"`,
 			},
 		},
 	}
+
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			g := NewWithT(t)
+			var buf bytes.Buffer
 
 			// Creates the output table
-			tbl := uitable.New()
+			tbl := tablewriter.NewWriter(&buf)
 
 			// Add row for the root object, the cluster, and recursively for all the nodes representing the cluster status.
 			addObjectRow("", tbl, tt.objectTree, tt.objectTree.GetRoot())
 
-			for i := range tt.expectPrefix {
-				g.Expect(tbl.Rows[i].Cells[0].String()).To(Equal(tt.expectPrefix[i]))
-			}
+			checkTableEquality(t, buf.String(), tt.expectPrefix[0], "Invalid Value in Table")
```

Comment:
1) To keep the gomega way of performing check consider writing a custom gomega matcher to check tables. Example for writing a custom gomega mtcher: https://github.com/kubernetes-sigs/cluster-api/blob/main/cmd/clusterctl/client/clusterclass_test.go#L222-L267

2) With the changes the behavior of the test seems to be changes. Now the test is comparing the full table instead of checking the prefixes. Lets keep the behavior of the tests the same. 
This can be solved by introducing the custom matcher. The matcher can take as input the final rendered table and a slice of the expected prefixes (you wont have to change the test cases in this case). In the matcher you can read one line of the table at a time (you can find helper functions for that in the `bufio` package) and then check to see if the line matches the expected prefix.

Reply:
It would be nice to have tests to compare the full table output as well. However, that should be its own PR and not part of this PR.

Reply:
okay Ill look into it